File: Java/1046.java

import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int tempInicial, tempFinal;
        int horasFinais;

        tempInicial = sc.nextInt();
        tempFinal = sc.nextInt();

        if (tempInicial < tempFinal) {
            horasFinais = tempFinal - tempInicial;

        }
        
        else {
            horasFinais = 24 - tempInicial + tempFinal;
        }

        System.out.printf("O JOGO DUROU %d HORA(S)\n", horasFinais);

    }
}

File: Python/1046.py
A , B = list (map(int,input().split()))

if A == B and B == A:
    print ("O JOGO DUROU 24 HORA(S)")
else:
    if A > B:
        tempJogo = (24 - A) + B
        print (f"O JOGO DUROU {tempJogo} HORA(S)")
    else:
        tempJogo = B - A   
        print (f"O JOGO DUROU {tempJogo} HORA(S)")     